benefits.md

```diff
@@ -33,13 +33,17 @@ Conferences count as working days (7.5h per day). Transport to conferences count
 
 Try to give notice at least 10-14 days before attending a conference.
 
-Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a bonus for each conference day:
+Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a reimbursal for each away day (conference and travel):
```

Comment:
Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance) and the conference allowance which accounts for all the days you are away at the conference, including travelling: